core/src/env/test_env.rs

```diff
@@ -120,6 +120,8 @@ pub struct TestEnvData {
     total_reads: Cell<u64>,
     /// The total number of writes to the storage.
     total_writes: u64,
+    /// Deposited events of the contract invocation.
+    events: Vec<u8>,
```

Comment:
Not quite sure this makes sense.

Right now, the only data we can send to `ext_deposit_event` is some `Vec<u8>`

This is because on the contracts module side we have defined an Event called `Contract(Vec<u8>)` which then gets placed inside of the `Events` enum with all the other possible events.

I could make a `ContractEvent` type of `Event::Contract(Vec<u8>)` or something, but I am not sure if this is right...